tests/fpga/streaming_memory_test.py

```diff
@@ -235,3 +979,34 @@ def test_streaming_and_composition():
     test_streaming_composition(None)
     test_streaming_composition_mapnests(None)
     test_streaming_and_composition(None)
+
```

Comment:
For the sake of CI testing times, can you reduce the number of tests?

You can leave them here, and decorate the test with `@pytest.mark.skip(reason="<put the reason here> ")`.
For example you can skip tests that use the same base type but different bitwidth (e.g., keep `int32` but skip `int8`/`int_16` ...)